sharktuner/setup.py

```diff
@@ -7,7 +7,7 @@
 import json
 import os
 
-from setuptools import setup
+from setuptools import setup  # type: ignore
```

Comment:
Could we add the typing info for these to our requirements? https://pypi.org/project/types-setuptools/